ansible/library/generate_golden_config_db.py

```diff
@@ -26,6 +26,14 @@
 TEMP_SMARTSWITCH_CONFIG_PATH = "/tmp/smartswitch.json"
 DUMMY_QUOTA = "dummy_single_quota"
 
+smartswitch_hwsku_config = {
```

Comment:
Hi @qiluo-msft Updated the code to make people to easy to udpate their port and inferface ip based on hwsku.